components/file_preview_modal/image_preview.tsx

```diff
@@ -7,8 +7,14 @@ import React from 'react';
 import {getFilePreviewUrl, getFileDownloadUrl} from 'mattermost-redux/utils/file_utils';
 
 import './image_preview.scss';
+import {FileInfo} from '@mattermost/types/files';
```

Comment:
nit : rearrange after
```suggestion
import {FileInfo} from '@mattermost/types/files';

import {getFilePreviewUrl, getFileDownloadUrl} from 'mattermost-redux/utils/file_utils';
```

Reply:
fixed